test/source/mock/google/strategies/send-message-strategy.ts

```diff
@@ -38,15 +38,15 @@ class PwdAndPubkeyEncryptedMessagesWithFlowCryptComApiTestStrategy implements IT
       const encryptedData = Buf.fromUtfStr(mimeMsg.text!);
       const decrypted = await MsgUtil.decryptMessage({ kisWithPp, encryptedData, verificationPubs: [] });
       expect(decrypted.success).to.be.true;
-      expect(decrypted.content!.toUtfStr()).to.contain('PWD and pubkey encrypted messages with flowcrypt.com/api');
+      expect(decrypted.content!.toUtfStr()).to.contain('PWD and pubkey encrypted messages with flowcrypt.com/shared-tenant-fes');
       expect(mimeMsg.bcc).to.be.an.undefined;
       expect(mimeMsg.to).to.be.an.undefined;
       expect((mimeMsg.headers.get('reply-to') as AddressObject).text).to.equal('First Last <flowcrypt.compatibility@gmail.com>, test@email.com');
     } else {
       expect(mimeMsg.text!).to.contain(`${senderEmail} has sent you a password-encrypted email`);
       expect(mimeMsg.text!).to.contain('Follow this link to open it');
       if (!mimeMsg.text?.match(/https:\/\/flowcrypt.com\/[a-z0-9A-Z]{10}/)) {
-        throw new HttpClientErr(`Error: cannot find pwd encrypted flowcrypt.com/api link in:\n\n${mimeMsg.text}`);
+        throw new HttpClientErr(`Error: cannot find pwd encrypted flowcrypt.com/shared-tenant-fes link in:\n\n${mimeMsg.text}`);
       }
```

Comment:
The link is expected to be different now, definitely not `/https:\/\/flowcrypt.com\/[a-z0-9A-Z]{10}/`. Try sending a password protected message. It will be something like `https://flowcrypt.com/shared-tentant-fes/message/...`

Reply:
@tomholub What will be exact url format?
I can't seem to get correct url format as live shared-tenant-fes is not available.

Reply:
> I can't seem to get correct url format as live shared-tenant-fes is not available.

It's available https://flowcrypt.com/shared-tenant-fes/

How are you attempting to test it?

Reply:
I tried with my gmail account and api says gmail.com client configuration is not available

Reply:
I'm asking about shared tenant password protected message url format though

Reply:
> I tried with my gmail account and api says gmail.com client configuration is not available

I see. I'll fix that. Then you can test with this, and you can then also exactly observe the url format.

In either case the url format will be exactly the same as when sending from flowcrypt.com domain, except that `fes.flowcrypt.com` becomes `flowcrypt.com/shared-tenant-fes`. The rest of the URL stays the same.

Reply:
Got it

Reply:
It's configured now for gmail.com: https://flowcrypt.com/shared-tenant-fes/api/v1/client-configuration?domain=gmail.com

It's not configured yet for custom user domains, waiting for https://github.com/FlowCrypt/flowcrypt-enterprise-on-prem/issues/4162 to be released first